storage/tianmu/handler/ha_tianmu.cpp

```diff
@@ -769,8 +769,9 @@ int ha_tianmu::open(const char *name, [[maybe_unused]] int mode, [[maybe_unused]
 }
 
 int ha_tianmu::free_share() {
+  DBUG_ENTER(__PRETTY_FUNCTION__);
   share_.reset();
-  return 0;
+  DBUG_RETURN(0);
```

Comment:
return value is `bool`, fix 
`0`--->`false`,
`1`--->`true`

Reply:
here return value is int